packages/docusaurus/src/webpack/client.ts

```diff
@@ -7,6 +7,7 @@
 import path from 'path';
 import {Configuration} from 'webpack';
 import merge from 'webpack-merge';
+import chalk from 'chalk';
```

Comment:
```suggestion
import chalk = require('chalk');
```
Hey great work so far, do we want to maybe import like I have suggested as its a fix shown here https://github.com/chalk/chalk/releases/tag/v3.0.0 and we are using `3.0.0`. Also we did it as part of this change https://github.com/facebook/docusaurus/pull/1985. 

I think its

Reply:
Good catch, thanks, fixed :heavy_check_mark: